### PR TITLE
feat: add annotations / questions for chart

### LINKED ISSUE #156: Add catalog annotations and questions
Updat ethe chart to include the annotation that are used when displaying the chart in the catalog. Also, add the `questions.yaml` so that the install can be customized.

### PR DESCRIPTION
**Pull request by richardcase**
**What this PR does / why we need it**:

Add the annotations for the chart to give a better experience in the app catalog.

Also, started to add questions to customize the install.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #156 

**Special notes for your reviewer**:

**Checklist**:

- [ ] squashed commits into logical changes
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

### CODE REVIEW

File: charts/rancher-turtles/questions.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  default: "true"
+  description: "Flag to enable or disable installation of the Cluster API Operator. If you disable this then you will need to manage the CAPI controllers yourself."
+  label: "Enable Cluster API Operator"
+  type: boolean
+  group: "Cluster API Operator"
+  show_subquestion_if: true
+  subquestions:
+  - variable: cluster-api-operator.cert-manager.enabled
+    default: "true"
+    description: "Flag to enable or disable installtion of cert-manager. If set to false then you will need to install cert-manager manually"
```
Comment (furkatgofurov7) [inline charts/rancher-turtles/questions.yml:12]:
```suggestion
    description: "Flag to enable or disable installation of cert-manager. If set to false then you will need to install cert-manager manually"
```

**Review by furkatgofurov7 — COMMENTED** [review]
Small nit:

**Comment by richardcase** [comment]
> This looks good @richardcase, what are outstanding steps to make this “non-draft”?

For me the main thing is to think about what questions we want to expose. I don't think we need to expose all the values...just the ones we think will be used by the majority of users.

**Review by salasberryfin — DISMISSED** [review]

File: charts/rancher-turtles/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Rancher Turtles Chart
+
+This chart installs the Rancher Turtles operator and optionally the Cluster API Operator using Helm.
+
+Chheckout the [documentation](https://docs.rancher-turtles.com) for further information.
```
Comment (furkatgofurov7) [inline charts/rancher-turtles/README.md:5]:
```suggestion
Checkout the [documentation](https://docs.rancher-turtles.com) for further information.
```

**Review by furkatgofurov7 — COMMENTED** [review]
nitty:

**Review by alexander-demicev — APPROVED** [review]

File: charts/rancher-turtles/README.md (hunk 2 above)
Comment (richardcase) [reply charts/rancher-turtles/README.md]: Thanks....changed.

**Review by furkatgofurov7 — APPROVED** [review]

**Review by salasberryfin — APPROVED** [review]

File: charts/rancher-turtles/questions.yml (hunk 1 above)
Comment (richardcase) [reply charts/rancher-turtles/questions.yml]: changed

**Pull request merged**